### ISSUE TITLE
Unable to use websockets on current release (1.1.2)

### ISSUE THREAD

**Issue by nvonpentz** [TrueFiEng/useDApp#906]
**Describe the bug**
I get an error trying to use an Ethers WebsocketProvider with useDapp config.
```
Uncaught Error: cannot set polling interval on WebSocketProvider (operation="setPollingInterval", code=UNSUPPORTED_OPERATION, version=providers/5.6.8)
    at Logger.makeError (index.ts:261:1)
    at Logger.throwError (index.ts:273:1)
    at set pollingInterval [as pollingInterval] (websocket-provider.ts:187:1)
    at provider.tsx:69:1
    at commitHookEffectListMount (react-dom.development.js:23150:1)
    at commitPassiveMountOnFiber (react-dom.development.js:24926:1)
    at commitPassiveMountEffects_complete (react-dom.development.js:24891:1)
    at commitPassiveMountEffects_begin (react-dom.development.js:24878:1)
    at commitPassiveMountEffects (react-dom.development.js:24866:1)
    at flushPassiveEffectsImpl (react-dom.development.js:27039:1)
```

**To Reproduce**
Try to pass a WebSocketProvider to the config:
```
const websocketsProvider = new providers.WebSocketProvider(`wss://mainnet.infura.io/ws/v3/${INFURA_PROJECT_ID}`)
const config = {
  readOnlyChainId: Mainnet.chainId,
  readOnlyUrls: {
    [Mainnet.chainId]: websocketsProvider
  },
}
```

**Software versions**
 - `useDapp` version (installed version of `@usedapp/core`) 1.1.2
 - Package manager (yarn, npm, or pnpm?)
 - Node version (`node --version`)

**Additional context**
I think this could be the reason - assuming all providers can have polling interval set.  That will throw for websocket providers
https://github.com/TrueFiEng/useDApp/blob/a7c6620ce8f9d7d82fa30f0caecd40b2f2fc6e7d/packages/core/src/providers/network/readonlyNetworks/provider.tsx#L67-L71

Potentially introduced in https://github.com/TrueFiEng/useDApp/pull/836.

I downgraded to 1.0.10 and it seems to work.

**Comment by rzadp** [comment]
Thanks @nvonpentz,
we will pick it up after the weekend, looks like not a difficult thing to fix.
Or if you would be willing to send a PR that would also be welcome.

**Comment by rzadp** [comment]
https://github.com/TrueFiEng/useDApp/pull/911 is on the way

**Comment by ItsShadowl** [comment]
> #911 is on the way

That doesn't fix it, i tested today and still get the same error
![image](https://user-images.githubusercontent.com/7571856/183998848-3fd0f19b-41f6-4c8e-82d3-f6e2f1c1eb1a.png)
I pushed a fix(#926) that completes the #911 fixes. Its just a probably overlooked fix.
